### ISSUE TITLE
Can we use megahit to assemble rna-seq data?

### ISSUE THREAD

**Issue by xiekunwhy** [voutcn/megahit#332]
Hi,

Can we use megahit as a denovo assembler to assemble rna-seq data?

Best,
Kun

**Comment by WJT0925** [comment]
I have the same question， and I test trinity and megahit using the same metatranscriptomes pair-data about 50G, and the result shows a longer N50 assembly by megahit, also use more data than trinity, and both two result mapping rate similar, BUSCO of megahit was 96.5%, for trinity was 98%, so I think it maybe ok for metatranscriptoms.

**Comment by aquaskyline** [comment]
Yes, megahit works with RNA-seq data, or any other ununified depth data.
